RFS/src/test/java/org/opensearch/migrations/bulkload/common/LuceneDocumentsReaderTest.java

```diff
@@ -164,41 +163,40 @@ public void ReadDocuments_ES5_Origin_AsExpected() throws Exception {
         // Use the LuceneDocumentsReader to get the documents
         var reader = LuceneDocumentsReader.getFactory(sourceResourceProvider).apply(luceneDir);
 
-        Flux<RfsLuceneDocument> documents = reader.readDocuments()
-            .sort(Comparator.comparing(doc -> doc.id)); // Sort for consistent order given LuceneDocumentsReader may interleave
+        Flux<RfsLuceneDocument> documents = reader.readDocuments();
 
         // Verify that the results are as expected
         StepVerifier.create(documents).expectNextMatches(doc -> {
-            String expectedId = "complexdoc";
+            String expectedId = "unchangeddoc";
             String actualId = doc.id;
 
-             String expectedType = "type1";
-             String actualType = doc.type;
+            String expectedType = "type2";
+            String actualType = doc.type;
 
-            String expectedSource = "{\"title\":\"This is a doc with complex history. Updated!\"}";
+            String expectedSource = "{\"content\":\"This doc will not be changed\nIt has multiple lines of text\nIts source doc has extra newlines.\"}";
             String actualSource = doc.source;
-            assertDocsEqual(expectedId, actualId, expectedType, actualType,
-                    expectedSource, actualSource);
+            assertDocsEqual(expectedId, actualId, expectedType, actualType, expectedSource, actualSource);
             return true;
         }).expectNextMatches(doc -> {
-            String expectedId = "unchangeddoc";
+            String expectedId = "updateddoc";
             String actualId = doc.id;
 
-             String expectedType = "type2";
-             String actualType = doc.type;
+            String expectedType = "type2";
+            String actualType = doc.type;
 
-            String expectedSource = "{\"content\":\"This doc will not be changed\nIt has multiple lines of text\nIts source doc has extra newlines.\"}";
+            String expectedSource = "{\"content\":\"Updated!\"}";
             String actualSource = doc.source;
-            assertDocsEqual(expectedId, actualId, expectedType, actualType, expectedSource, actualSource);
+            assertDocsEqual(expectedId, actualId, expectedType, actualType,
+                    expectedSource, actualSource);
             return true;
         }).expectNextMatches(doc -> {
-            String expectedId = "updateddoc";
+            String expectedId = "complexdoc";
             String actualId = doc.id;
 
-             String expectedType = "type2";
+             String expectedType = "type1";
              String actualType = doc.type;
 
-            String expectedSource = "{\"content\":\"Updated!\"}";
+            String expectedSource = "{\"title\":\"This is a doc with complex history. Updated!\"}";
             String actualSource = doc.source;
             assertDocsEqual(expectedId, actualId, expectedType, actualType,
```

Comment:
just verifying that these orders changed because we are now sorting segments & docs?

Reply:
Correct